training/src/anemoi/training/diagnostics/mlflow/logger.py

```diff
@@ -237,7 +237,14 @@ def _remove_csi(line: bytes) -> bytes:
                 logfile.write(cleaned_line.decode())
 
             logfile.flush()
-        self.experiment.log_artifact(self.run_id, str(self.file_save_path))
+
+        # This try/except is for AML logging
+        # typically terminal_output.txt gets updated by overwriting, but this is not allowed on AML
+        # since this happens tons of times, just silently pass if it fails
+        try:
+            self.experiment.log_artifact(self.run_id, str(self.file_save_path))
+        except:
+            pass
```

Comment:
Nothing intelligent to add here, just wanted to say that my linter flagged that this pattern is preferred to `try`/`except`/`pass`:

```python
from contextlib import suppress


with suppress(Exception):
    self.experiment.log_artifact(self.run_id, str(self.file_save_path))
```

I do think we could think of something better here though; maybe logging a new file per-epoch? Or at least shipping this feature with no terminal logging for now, since we have the logs already on AzureML?